lms/djangoapps/instructor/views/api.py

```diff
@@ -1361,6 +1361,30 @@ def get_exec_summary_report(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+@require_finance_admin
```

Comment:
actually, we should not require "finance_admin" rights here. Any course staff should be able to do this. I'll remove.